src/Nethermind/Nethermind.Consensus/Validators/TxValidator.cs

```diff
@@ -282,3 +310,42 @@ public sealed class SignatureTxValidator : BaseSignatureTxValidator
     public static readonly SignatureTxValidator Instance = new();
     private SignatureTxValidator() { }
 }
+
+public sealed class SetCodeTxValidator : ITxValidator
```

Comment:
It seems that this validation is being omitted in the BlockProductionTransactionPicker.

Reply:
I dont see the TxValidator checks being used there, maybe because the logic is that picked tx comes from `TxPool`?

Reply:
> Yes, but I think Shutter omits TxPool validation. This should be fixed on the Shutter branch though. cc: @Marchhill

Yes, it does but it has its own filter here:
https://github.com/NethermindEth/nethermind/blob/master/src/Nethermind/Nethermind.Shutter/ShutterTxFilter.cs